CI/harvester-installer/roles/jenkins/tasks/install_vagrant_on_Debian.yml

```diff
@@ -1,7 +1,10 @@
 ---
+# NOTE: this is strange...I have no idea why the `-1` on the package has to be present now...but it seems they changed
+# the way they are serving the packages... idk... here's to being "optimistic" that they won't change their file
+# serving pattern again in the future...
 - name: jenkins | install_vagrant_on_Debian | Install vagrant package
   apt:
-    deb: https://releases.hashicorp.com/vagrant/{{ LATEST_VAGRANT_VERSION }}/vagrant_{{ LATEST_VAGRANT_VERSION }}_x86_64.deb
+    deb: https://releases.hashicorp.com/vagrant/{{ LATEST_VAGRANT_VERSION }}/vagrant_{{ LATEST_VAGRANT_VERSION }}-1_amd64.deb
```

Comment:
Is this going to the version, then {{version_number}}-1, or did the URL change somehow? It almost looks like that is a second file that had a `-1` added to it to prevent filename collision.

Reply:
The `-1` is this new weird thing that I mentioned in the code-comment above this:
```
# NOTE: this is strange...I have no idea why the `-1` on the package has to be present now...but it seems they changed
# the way they are serving the packages... idk... here's to being "optimistic" that they won't change their file
# serving pattern again in the future..
```
Like their file serving seems to have changed with Hashicorp.
For instance, if you notice here:
https://releases.hashicorp.com/vagrant/2.3.4/
They append the `-1` to the end of the deb package, which is what we need.
And they seem to have also done that for:
https://releases.hashicorp.com/vagrant/2.3.3/
So I'm "guessing" that moving forward this will be Hashicrops file serving way of distributing the packages